skycoin/skywire
# Cleanup util

Did you run `make format && make check`?
yes

Fixes #	

 Changes:	
- Deleted packages alignment, atomicbool and stdio
- Used netutil from skywire-utilities

How to test this PR:
1. Build visor `make build`
2. Run  visor `./skywire-visor -c skywire-config.json`
3. Check if everything is working as expected
